package.json

```diff
@@ -17,6 +17,10 @@
     "test:watch": "jest --watch --runInBand",
     "test:docker": "docker compose -f docker-compose.test.yml up -d --build",
     "test:ci": "jest --ci --runInBand",
+    "_TEST CYPRESS SCRIPTS": "===",
```

Comment:
Should probably include scripts to run tests (headlessly) locally to streamline testing